yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Unused as we don't have package.json in the root, but @euanmillar we could consider adding some "start-all" type of a script to the root of the package.. just haven't had the need yet.